modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/40_token_filters.yml

```diff
@@ -662,6 +662,22 @@
     - match:  { tokens.0.token: baz }
     - match:  { tokens.1.token: baz }
 
+---
+"stemmer_override file access":
+    - do:
+        catch: request
+        indices.create:
+          index: test
+          body:
+            settings:
+              analysis:
+                filter:
+                  my_stemmer_override:
+                    type: stemmer_override
+                    rules_path: "jvm.options"
+    - match: { status: 500 }
```

Comment:
Could we catch this in the same way FileWatcher was modified, to result in a 400 instead of a 500?

Reply:
I've changed it to a 400 exit code